custom/icds/messaging/indicators.py

```diff
@@ -108,6 +127,37 @@ def location_names_from_user_id(self, user_ids):
         return {self.awc_locations[l] for l in loc_ids}
 
 
+class AWWAggregatePerformanceIndicator(AWWIndicator):
+    template = 'ls_aggregate_performance.txt'
+
+    def get_value_from_fixture(self, fixture, attribute):
+        xpath = './rows/row[@is_total_row="False"]'
+        rows = fixture.findall(xpath)
+        for row in rows:
+            owner_id = row.find('./column[@id="owner_id"]')
+            if owner_id.text == self.user.sql_location.name:
```

Comment:
Not if we want to use mobile ucr to handle the filtering and generation.

the column says `owner_id` but it's actually transformed into a name. ex https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/reports/ls_awc_days_open.json#L42-L56

We could possibly change the report to include two columns, but I don't know how feasible that is